bindata/bootkube/manifests/daemonset-kube-controller-manager.yaml

```diff
@@ -43,14 +42,15 @@ spec:
           name: service-account-private-key
         - mountPath: /var/run/secrets/cluster-signing-ca
           name: cluster-signing-ca
-        - mountPath: /var/run/secrets/client-ca
+        - mountPath: /var/run/configmaps/client-ca
           name: client-ca
       nodeSelector:
         node-role.kubernetes.io/master: ""
       tolerations:
       - key: node-role.kubernetes.io/master
         operator: Exists
         effect: NoSchedule
+      hostNetwork: true
```

Comment:
@sttts this might not be needed after all if the networking operator is able to reach the API server after we deploy the kube-proxy

Reply:
Probably the phase-1 controller manager creates all necessary service accounts and what not. Then this comes up even without host networking. Have the slight feeling that this might work most often, but it's more a conincidence.